frontend/src/components/MeetingRoom/Toolbar/Toolbar.tsx

```diff
@@ -57,12 +60,14 @@ const Toolbar = ({
   const isReportIssueEnabled = import.meta.env.VITE_ENABLE_REPORT_ISSUE === 'true';
   const isViewingScreenShare = subscriberWrappers.some((subWrapper) => subWrapper.isScreenshare);
   const isScreenSharePresent = isViewingScreenShare || isSharingScreen;
+  const isSmallViewport = useIsSmallViewport();
   const handleLeave = useCallback(() => {
     if (!disconnect) {
       return;
     }
     disconnect();
   }, [disconnect]);
+  const [openEmojiGridDesktop, setOpenEmojiGridDesktop] = useState<boolean>(false);
```

Comment:
On desktop, we wouldn't necessarily need to have this slice of state declared at an upper level, but it's messier to have conditional logic at the component level rather than treating desktop and "mobile" views the same, imo